locations/container/README.md

```diff
@@ -0,0 +1,30 @@
+
+# Kubernetes Location
+
+Brooklyn Container Location has an extensive support for Kubernetes deployments
+In particular, it supports
+
+- KubernetesResource
+- KubernetesHelmChart
+- KubernetesContainer
+
+## Kubernets Helm Chart
+
+Here's an example of an Helm based blueprint 
+
+```YAML
+location:
+  kubernetes:
+    endpoint: https://localhost:6443
+    kubeconfig: /home/user/.kube/config
+services:
+- type: org.apache.brooklyn.container.entity.kubernetes.KubernetesHelmChart
+  name: jenkins-helm
+  chartName: jenkins
```

Comment:
is there a way to load a chart embedded in a BOM?  that would be handy to document here.

Reply:
Not sure yet, solving the dependency hell is still onlgoing.